special-use/special-use-suds-mapping.json

```diff
@@ -6,97 +6,97 @@
       "applicant-info": {
         "type": "object",
         "properties": {
-      		"firstName": { "type": "string", "suds-application": "Contacts", "suds-field": "First Name"},
-      		"lastName": {	"type": "string", "suds-application": "Contact", "suds-field": "Last Name" },
+      		"firstName": { "type": "string", "basic-api-field" : "FIRST_NAME"},
+      		"lastName": {	"type": "string", "basic-api-field" : "LAST_NAME" },
```

Comment:
Ah, so this is reflected in line 91 and 92. The request can either include a CONT_ID or this applicant-info block. If it does include the applicant-info block, then all these fields are required.